src/System.Net.Mail/tests/Functional/AttachmentTest.cs

```diff
@@ -32,6 +32,23 @@ public void ConstructorNullName()
             Assert.Equal(null, attach.Name);
         }
 
+        [Fact]
+        public void ConstructorPathName()
```

Comment:
Does this new test fail without your product source fixes?  Just want to make sure that the test is actually testing a fix that uses that product source change.